Libs/MRML/DisplayableManager/vtkThreeDViewInteractorStyle.cxx

```diff
@@ -792,6 +783,56 @@ void vtkThreeDViewInteractorStyle::SetModelDisplayableManager(
 }
 
 //----------------------------------------------------------------------------
+void vtkThreeDViewInteractorStyle::RotateToPosterior(bool inverse)
```

Comment:
Sure. Do we want this in `vtkThreeDViewInteractorStyle` as well as `qMRMLThreeDView`?